x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -387,9 +390,9 @@ export interface BulkActionsObjectProp {
 }
 
 export interface AlertsTableProps {
+  ownerPluginId: string;
```

Comment:
nit: To avoid confusion with the case's owner let's call the prop `pluginId` or similar.